src/utils/oddsCalculator.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * @description Returns decimal odds given numerator and denominator of fractional odds.
+ * @arg {number} numerator
+ * @arg {number} denominator
+ * @arg {number} decimalPlaces
+ * @example
+ * // return 1.33
+ * convertFractionalToDecimal(1, 3)
+ */
+export const convertFractionalToDecimal = (numerator, denominator, decimalPlaces = 2) => {
+  const result = (numerator + denominator) / denominator;
```

Comment:
The calculation here is wrong it should be:

const result = numerator / denominator. 

Adding the denominator to the numerator effectively gives you the answer + 1

Reply:
The calculation is correct. I'm on my phone rn but Google it and you will see 1/3 on a betting website would be 1.33 decimal.

Reply:
Otherwise you would lose money on odds of 1/3. 0.33 * your stake makes no sense.